tembo-operator/src/cloudnativepg/hibernate.rs

```diff
@@ -129,6 +138,23 @@ pub async fn reconcile_cluster_hibernation(cdb: &CoreDB, ctx: &Arc<Context>) ->
         }
     });
 
+    let scheduled_backup_suspend_status = scheduled_backup.spec.suspend.unwrap_or_default();
+    let scheduled_backup_value = cdb.spec.stop;
+    let patch_scheduled_backup_spec = json!({
+        "spec": {
+            "suspend": scheduled_backup_value
+        }
+    });
+
+    // Check if scheduled_backup_suspend_status=false and cdb.spec.stop=true.  If so then patch the scheduled backup suspend status to true
+    if !scheduled_backup_suspend_status && cdb.spec.stop {
+        patch_scheduled_backup_merge(cdb, ctx, patch_scheduled_backup_spec.clone()).await?;
+        info!(
+            "Toggled scheduled backup suspend of {} to '{}'",
+            name, scheduled_backup_value
+        );
+    }
+
```

Comment:
This part should handle the case where instances are already paused but the scheduled backup is not suspended.
 
This will not un-suspend the backups though.